scripts/get_maintainer.py

```diff
@@ -403,12 +405,16 @@ def _print_areas(areas):
 \tcollaborators: {}
 \tinform: {}
 \tlabels: {}
+\ttests: {}
+\ttags: {}
 \tdescription: {}""".format(area.name,
                             area.status,
                             ", ".join(area.maintainers),
                             ", ".join(area.collaborators),
                             ", ".join(area.inform),
                             ", ".join(area.labels),
+                            ", ".join(area.tests),
+                            ", ".join(area.tags),
```

Comment:
I'm a bit confused by the tags here, is this a transient? You are adding handling, then adding some tags to the maintainer file, then removing the tag and only using tests or am I missing something?

Reply:
yes, started with tags but realized those are very random and inconsistent and do not provide direct relation and decided to look into test identifiers instead which are more accurate.